src/MagnumPlugins/KtxImporter/Test/generate.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+
+# https://github.com/KhronosGroup/KTX-Software v4.0.0
+
+toktx version1.ktx white.png
+
+toktx --t2 --nometadata orientation-empty.ktx2 pattern.png
+#toktx --t2 orientation-rd.ktx2 pattern.png
+#toktx --t2 --lower_left_maps_to_s0t0 orientation-ru.ktx2 pattern.png
+
+toktx --t2 --swizzle rgb1 swizzle-identity.ktx2 pattern.png
+toktx --t2 --target_type RGBA --swizzle rgb1 swizzle-unsupported.ktx2 pattern.png
+
+# swizzled data, same swizzle in header
+# data should come out normally after the importer swizzles it
+toktx --t2 --input_swizzle bgra --swizzle bgr1 bgr-swizzle-bgr.ktx2 pattern.png
+toktx --t2 --target_type RGBA --input_swizzle bgra --swizzle bgra bgra-swizzle-bgra.ktx2 pattern.png
+
+# swizzled header
+# with patched swizzled vkFormat data should come out normally because both cancel each other out
+toktx --t2 --swizzle bgr1 swizzle-bgr.ktx2 pattern.png
+toktx --t2 --target_type RGBA --swizzle bgra swizzle-bgra.ktx2 pattern.png
+
+# swizzled data
+# with patched swizzled vkFormat data should come out normally
+toktx --t2 --input_swizzle bgra bgr.ktx2 pattern.png
+toktx --t2 --target_type RGBA --input_swizzle bgra bgra.ktx2 pattern.png
+
+# 2D
+toktx --t2 rgb.ktx2 pattern.png
+toktx --t2 --target_type RGBA rgba.ktx2 pattern.png
+
+# manual mipmaps
+toktx --t2 --mipmap rgb-mipmaps.ktx2 @@pattern-mips.txt
+
```

Comment:
`toktx` seems to choke on creating 3D images, not sure if I'm doing something wrong or that path is completely untested. I was thinking I'd just generate a 2D layered image and patch the header to make it look like 3D since the data layout is the same, and `toktx` doesn't support 3D layered images. For those we'd need something else, not sure if using `magnum-imageconverter` is considered cheating 👀

Reply:
> For those we'd need something else

For EXR I was generating the ground truth data with `./OpenExrImageConverterTest --save-diagnostics .` and then using those in `OpenExrImporterTest` (yeah and then cross-checking that these images are still readable by the official tools). The `imageconverter` doesn't even have 3D image support right now, I'd need to add that there first, going through the converter test seems like a lot easier option :)